File: Python/Processor.py
class Processor:
    # kelas yang digunakan untuk mengimplementasikan Processor
    # private attribute dari kelas disk
    __name = ""
    __price = 0

    def __init__(self, name="", price=0):
        # konstruktor
        self.__name = name
        self.__price = price

    # mengeset nilai atribut name
    def setName(self, name):
        self.__name = name

    # mengembalikan nilai atribut name
    def getName(self):
        return self.__name

    # mengeset nilai atribut price
    def setPrice(self, price):
        self.__price = price

    # mengembalikan nilai atribut price
    def getPrice(self):
        return self.__price

    # menampilkan atribut Processor
    def printProcessor(self):
        print("Processor name : " + self.getName())
        print("Processor Price : " + str(self.getPrice()))

File: PHP/Processor.php
<?php
class Processor
{
    // private atribute dari kelas Processor
    private $name;
    private $price;

    // konstruktor
    function __construct($name, $price)
    {
        $this->name = $name;
        $this->price = $price;
    }

    // mengeset nilai atribut name
    function setName($name)
    {
        $this->name = $name;
    }

    // mengembalikan nilai atribut name
    function  getName()
    {
        return $this->name;
    }

    // mengeset nilai atribut price
    function setPrice($price)
    {
        $this->price = $price;
    }

    // mengembalikan nilai atribut price
    function getPrice()
    {
        return $this->price;
    }

    // menampilkan atribut Processor
    function printProcessor()
    {
        echo "Processor name : " . $this->getName() . "<br>";
        echo "Processor Price : " . $this->getPrice() . "<br>";
    }

    function __destruct()
    {
    }
}
